package.json

```diff
@@ -2,33 +2,37 @@
   "name": "@lob/ui-components",
   "version": "0.0.176",
   "engines": {
-    "node": ">=14.16.1",
-    "npm": ">=7.13.0"
+    "node": ">=14.18.2",
+    "npm": ">=8.3.0"
+  },
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/ui-components.umd.js",
+  "module": "./dist/ui-components.es.js",
+  "exports": {
+    ".": {
+      "import": "./dist/ui-components.es.js",
+      "require": "./dist/ui-components.umd.js"
+    }
   },
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "build-library": "vue-cli-service build --target lib --name ui-components src/main.js",
+    "build": "vite build",
     "build-storybook": "build-storybook",
-    "lint": "npm run lint:js && npm run lint:css",
-    "lint-fix": "vue-cli-service lint --fix",
     "lint:css": "stylelint 'src/**/*.vue'",
     "lint:js": "eslint --ext .js,.vue .",
-    "test": "npm run test:ci",
-    "test:ci": "npm run test:unit",
-    "test:unit": "vue-cli-service test:unit",
+    "lint": "npm run lint:js && npm run lint:css",
+    "lint-fix": "npm run lint --- --fix",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
```

Comment:
I forgot to point this out: if you want to have the tests run after every code change (and only the tests on the file you just changed) you can run this script.  It's super fast so it shouldn't bog down your coding and will give you quick feedback if your change just broke a test.